miqi/agent/tools/graph_render.py

```
@@ -290,16 +290,19 @@ def _layout(graph: dict[str, Any]) -> dict[str, Any]:
             }
         )
 
+    # 节点实际高度映射（多行标题节点 h > _NODE_H_BASE，边须连接垂直中心）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)